### ISSUE TITLE
c4 and cockroaches

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#13078]
(WEB REPORT BY: pvak REMOTE: 172.93.109.202:7777)
>Revision (Should be above if you're viewing this from ingame!)

>General description of the issue
cockroaches take no damage when you stick c4 on them and blow it up

>What you expected to happen
exposive cockroaches

>What actually happened
cockroaches can survive explosions from c4 stuck on em

>Steps to reproduce if possible
plant c4 on cockroach and look how he is resistant to explosions

**Comment by Kammerjunk** [comment]
I think this might be intended, isn't it?

**Comment by jknpj** [comment]
how the fuck do you even stuck a c4 in a roach

**Comment by Probe1** [comment]
very carefully

**Comment by Kammerjunk** [comment]
I would imagine this is what ex_act() in the cockroach file prevents. They'll just try flying away for 10 seconds.
They'll also survive nuclear explosions and temperatures up to and including infinity.

**Comment by Kitfox88** [comment]
remove controlled roach for c4 shenanigans as tator item when

**Comment by juanototo2** [comment]
>>>>>>>>>>>>>>>>>>>>They'll also survive nuclear explosions and temperatures up to and including infinity.

**wait, WHAT?**

**Comment by Pathid** [comment]
It's true in the game. (Except INFINITY isn't really infinite, it's just some ridiculously high number.)

**Comment by Exxion** [comment]
You are correct that the INFINITY define is not infinite, but BYOND does, in fact, allow a number to be infinite
It's called 1.#Inf or something like that
There was briefly a bug that allowed temperatures to get that high, too
